static/bidder-params/cwire.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "CWire Adapter Params",
+  "description": "A schema which validates params accepted by the CWire adapter",
+  "type": "object",
+  "properties": {
+    "placementId": {
+      "type": "integer",
+      "description": "An ID which identifies this placement of the impression"
+    },
+    "pageId": {
+      "type": "integer",
+      "description": "An ID which identifies the site selling the impression"
+    },
+    "cwcreative": {
+      "type": "string",
+      "description": "An CWire ID of the creative that we want to show"
+    },
+    "cwdebug": {
+      "type": "boolean",
+      "description": "Enable CWire debug mode"
+    },
+    "cwfeatures": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "description": "A string array of CWire features"
+    }
+  },
+  "required": []
```

Comment:
In the docs I see that all these parameters are said to be required but here they aren't. Is that intentional?

Reply:
Yeah, it's intentional. We also have Prebid.js adapter and it does require those fields (PBS adapter doesn't).
Anyway, my colleague is currently working on Prebid.js adapter and soon we should have those requirements aligned between the two.

Reply:
@MatejaMaric Sounds good. In this case it sounds like you want the parameters to be optional in both PBJS and Prebid Server so this sounds fine but if you'd be looking at making any of these required any time in the future then we'd recommend making all of them required to start with and then move to making some of them optional based on your need. Because it's easier to go from required to optional but difficult to do the other way round without breaking existing setups of Publishers.